dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -857,3 +857,4 @@ CONTENT_EDITOR2_ENABLED=false
 LOCALIZATION_ENHANCEMENTS_ENABLED=false
 
 STARTER_BUILD_VERSION=${starter.deploy.version}
+FEATURE_FLAG_CONTENT_ANALYTICS=false
```

Comment:
For general readability and discoverability, should this be moved up under the `## Telemetry` section ~line 850